src/containers/help/HelpContainer.ts

```diff
@@ -0,0 +1,40 @@
+import { FC, useEffect, useState } from 'react';
+import { HelpApi } from '../../api/HelpApi';
+import { ComponentOrChildrenFn, renderComponentOrChildrenFn } from '../../utils/containers/ComponentOrChildrenFn';
+
+export interface HelpContainerProvided {
+  helpTextMd: string | undefined;
+  isLoading: boolean;
+}
+
+interface HelpContainerProps {
+  helpApi: HelpApi;
+}
+
+const HelpContainer: FC<HelpContainerProps & ComponentOrChildrenFn<HelpContainerProvided>> = ({ helpApi, ...rest }) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [helpTextMd, setHelpTextMd] = useState<string>();
+
+  const requestHelp = async () => {
+    try {
+      setIsLoading(true);
+      const helpText = await helpApi.getHelpTextMd();
+      setHelpTextMd(helpText);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    requestHelp();
+  }, []);
+
+  const provided: HelpContainerProvided = {
+    helpTextMd,
+    isLoading,
+  };
+
+  return renderComponentOrChildrenFn(rest, provided);
```

Comment:
I am reserved to passing `...rest` to the view. At this moment typescript isn't performing type checking well enough and the contract between the container and view isn't explicit.

Reply:
You mean that Typescript ignores what's in `rest` by having it casted to `any` or similar? Because what I see is that it effectively treats rest as `type Rest = Omit(Whole, 'expicitAttr1' | 'explicitAttr2' | ...)`. Maybe I miss some point here...